src/yaml/from_yaml.cr

```diff
@@ -8,6 +8,13 @@ def Object.from_yaml(string_or_io) : self
   end
 end
 
+def Object.from_yaml?(string_or_io) : self | Bool
```

Comment:
What's the purpose of this method? It's behaviour seems very unexpected. Usually `?`-methods don't raise an error but return `nil` or the regular value.